bspump/elasticsearch/connection.py

```diff
@@ -256,18 +253,21 @@ def get_url(self):
 	def get_session(self):
 		return aiohttp.ClientSession(auth=self._auth, loop=self.Loop)
 
-	def consume(self, index, _id, data, bulk_class=ElasticSearchBulk):
+	def consume(self, index, data, bulk_class=ElasticSearchBulk):
```

Comment:
`data` is actually `data feeder` ... right?
Adjust names accordingly please.

Reply:
@ateska It is actually the generator output of data feeder, i. e. output = data_feeder(event) ... What is the best name here?

Reply:
`data_feeder_generator`